xonsh/tools.py

```diff
@@ -23,10 +23,10 @@
 import contextlib
 import ctypes
 import datetime
-from distutils.version import LooseVersion
 import functools
 import glob
 import itertools
+import operator
```

Comment:
Hey @msoedov -- I'd prefer if we didn't touch lines unrelated to actual issues just to alphabetize or similar, it makes git blame/praise less useful for tracking down issues.